File: demo_java/src/main/java/com/techcentrix/sdk/demo_java/ui/DemoActivity.java
package com.techcentrix.sdk.demo_java.ui;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.techcentrix.sdk.demo_java.R;

public class DemoActivity extends AppCompatActivity {

    public DemoActivity() {
        super(R.layout.activity_demo);
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setSupportActionBar(findViewById(R.id.actionBarToolbar));
    }
}

File: demo_kotlin/src/main/java/com/techcentrix/sdk/demo_kotlin/ui/DemoActivity.kt
package com.techcentrix.sdk.demo_kotlin.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.techcentrix.sdk.demo_kotlin.R
import kotlinx.android.synthetic.main.activity_demo.*

class DemoActivity : AppCompatActivity(R.layout.activity_demo) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setSupportActionBar(actionBarToolbar)
    }
}
